apps/easypid/app.config.js

```diff
@@ -1,180 +1,59 @@
-const { version } = require('./package.json')
-
-const APP_VARIANT = process.env.APP_VARIANT || 'production'
-
-const variants = {
-  development: {
-    bundle: '.dev',
-    name: ' (Dev)',
-    mediatorDid: 'did:web:mediator.dev.paradym.id',
-  },
-  preview: {
-    bundle: '.preview',
-    name: ' (Preview)',
-    mediatorDid: 'did:web:mediator.paradym.id',
-  },
-  production: {
-    bundle: '',
-    name: '',
-    mediatorDid: 'did:web:mediator.paradym.id',
-  },
+const { version: easypidVersion } = require('./package.json')
+const { version: paradymVersion } = require('../paradym/package.json')
+const { createBaseConfig } = require('./base.app.config')
+
+const mediatorDids = {
+  development: 'did:web:mediator.dev.paradym.id',
+  preview: 'did:web:mediator.paradym.id',
+  production: 'did:web:mediator.paradym.id',
 }
 
-const variant = variants[APP_VARIANT]
-
-if (!variant) {
-  throw new Error(`Invalid variant provided: ${process.env.APP_VARIANT}`)
+const APP_CONFIGS = {
+  FUNKE_WALLET: createBaseConfig({
+    name: 'Funke Wallet',
+    scheme: 'id.animo.ausweis',
+    icon: './assets/funke/icon.png',
+    adaptiveIcon: './assets/funke/adaptive-icon.png',
+    splash: './assets/funke/splash.png',
+    slug: 'ausweis-wallet',
+    version: easypidVersion,
+    bundleId: 'id.animo.ausweis',
+    additionalInvitationSchemes: ['id.animo.ausweis', 'haip'],
+    associatedDomains: ['funke.animo.id'],
+    projectId: '28b058bb-3c4b-4347-8e72-41dfc1dd99eb',
+  }),
+
+  PARADYM_WALLET: createBaseConfig({
+    name: 'Paradym Wallet',
+    scheme: 'id.animo.paradym',
+    icon: './assets/paradym/icon.png',
+    adaptiveIcon: './assets/paradym/adaptive-icon.png',
+    splash: './assets/paradym/splash.png',
+    slug: 'paradym-wallet',
+    version: paradymVersion,
+    bundleId: 'id.paradym.wallet',
+    additionalInvitationSchemes: ['didcomm'],
```

Comment:
We should probably also add `id.animo.paradym` here to allow for correct deeplinking

Reply:
Also we need to extract it and use it here: https://github.com/animo/paradym-wallet/blob/e9b754122ba9e872de4e41e89d263a74157e1917/apps/easypid/src/features/receive/FunkeCredentialNotificationScreen.tsx#L53

Reply:
Seems we can access it like this: https://docs.expo.dev/versions/latest/sdk/constants/#platformmanifest